nistats/reporting/report_template.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
```

Comment:
That was my original idea, and ultimate plan. But I want to get the other things working completely first. I will do that last.
Also t makes harder to see the layout of the final page, so last thing on the list

Reply:
We always knew that was gonna happen, but we wanted to see how far templating using python alone can take us and I have a few ideas about how this can be made even better. I am gonna try that first.